src/strings/en-gb.json

```
@@ -923,7 +923,7 @@
     "LabelMatchType": "Match type",
     "LabelManufacturerUrl": "Manufacturer URL",
     "LabelLoginDisclaimerHelp": "A message that will be displayed at the bottom of the login page.",
-    "LabelLockItemToPreventChanges": "Lock this item to prevent future changes",
+    "LabelLockItemToPreventChanges": "Lock this item to prevent future metadata changes",
```

Only `en-us.json` can be changed manually. For other languages use [Weblate](https://translate.jellyfin.org/).
```suggestion
    "LabelLockItemToPreventChanges": "Lock this item to prevent future changes",
```